config/subscriptions/CanadaESLZ-main/pubsec/LandingZones/DevTest/82f7705e-3386-427b-95b7-cbed91ab29a7_healthcare_canadacentral.json

```diff
@@ -94,11 +94,14 @@
               "aadAuthenticationOnly":false
             }
           },
-        "synapse": {
+          "synapse": {
             "value": {
-                "username": "azadmin"
+              "aadAuthenticationOnly": true,
+              "aadLoginName": "az.admins",
+              "aadLoginObjectID": "065a3306-1466-465d-ae31-bc7be9e324cc",
```

Comment:
Can you change the guid to `e0357d81-55d8-44e9-9d9c-ab09dc710785` ?

Reply:
done